code/modules/games/52card.dm

```diff
@@ -6,45 +6,91 @@
 /obj/item/deck/cards
 	name = "deck of cards"
 	desc = "A simple deck of playing cards."
-	icon_state = "deck"
+	icon_state = "deck_nanotrasen_full"
+	deck_style = "nanotrasen"
 
-/obj/item/deck/cards/New()
-	..()
-	for(var/suit in list("spades","clubs","diamonds","hearts"))
+/obj/item/deck/cards/build_deck()
+	for(var/suit in list("Spades","Clubs","Diamonds","Hearts"))
+		var/card_appearance
 		var/colour
-		if(suit == "spades" || suit == "clubs")
-			colour = "black_"
-		else
-			colour = "red_"
-
-		for(var/number in list("ace","two","three","four","five","six","seven","eight","nine","ten"))
-			cards += new /datum/playingcard("[number] of [suit]", "[colour]num")
+		var/rank
+		if(simple_deck)
+			if(suit in list("Spades","Clubs"))
+				colour = "black"
+			else
+				colour = "red"
+		for(var/number in list("Ace","2","3","4","5","6","7","8","9","10","Jack","Queen","King"))
+			if(simple_deck)
+				if(number in list("Jack","Queen","King"))
+					rank = "col"
+				else
+					rank = "num"
+				card_appearance = "sc_[colour]_[rank]_[deck_style]"
+			else
+				card_appearance = "sc_[number] of [suit]_[deck_style]"
```

Comment:
```suggestion
		for(var/number in list("Ace","2","3","4","5","6","7","8","9","10","Jack","Queen","King"))
			if(!simple_deck)
				card_appearance = "sc_[number] of [suit]_[deck_style]"
				continue
			    
			if(number in list("Jack","Queen","King"))
				rank = "col"
			else
				rank = "num"
			card_appearance = "sc_[colour]_[rank]_[deck_style]"

				
```

Slight nitpick but might make it a little clearer

Reply:
If you continue then you completely skip the part where the datum is created.

Reply:
I don't think so? Continue should just skip to the next iteration in the current loop.